learning/step_by_step/your_first_game.rst

```diff
@@ -316,6 +323,8 @@ create any number of independent mobs in the game.
 Node Setup
 ~~~~~~~~~~
 
+Click Scene -> New Scene and we'll create the Mob!
```

Comment:
I don't like the "!" here.  How about:

Click Scene -> New Scene to begin creating the Mob.